logical/framework/openapi.go

```diff
@@ -158,7 +158,7 @@ type OASSchema struct {
 	Default          interface{}           `json:"default,omitempty"`
 	Example          interface{}           `json:"example,omitempty"`
 	Deprecated       bool                  `json:"deprecated,omitempty"`
-	Required         bool                  `json:"required,omitempty"`
+	Required         []string              `json:"required,omitempty"`
```

Comment:
Would it be possible to add a comment somewhere near here about what would go in the string array? By reading the code where it's used, I can figure out that it's the field names. Or maybe it could be renamed to something even more explicit like `RequiredFieldNames`.

Reply:
I'll add a comment. This part of OpenAPI is annoying b/c the way they handle required is different between requestBody and parameters. I'm trying to keep the field names identical to their JSON names, though.